core/src/main/java/com/bumble/appyx/core/navigation/RoutingKey.kt

```diff
@@ -1,4 +1,4 @@
-package com.bumble.appyx.core.routing
+package com.bumble.appyx.core.navigation
```

Comment:
Rename RoutingKey to something else, as it no longer makes sense. `NavKey`/`NavigationKey`

Reply:
also, rename any variables called `routingKey` to `navKey` (or just `key` if posssible)

Reply:
You'll also most likely need to do a second pass of updates in the documentation

Reply:
Maybe a separate PR, it's a large changeset in itself.

Reply:
happy if you want to do it in a separate PR, however I would suggest avoiding renaming them in two different releases.